### PR TITLE
PWX-29462: Add storkctl command for failover

### PR DESCRIPTION
**Pull request by dgoel-px**
What type of PR is this?
Improvement

What this PR does / why we need it:
This PR adds a new storkctl command storkctl perform failover that uses the Action CRD to kick off the failover process.

Does this PR change a user-facing CRD or CLI?:
Yes. The change is needed as the previous storkctl activate migration command has an unobvious meaning and this change is part of the bigger effort to simplify the DR use.

Is a release note needed?:
No

Does this change need to be cherry-picked to a release branch?:
No

### CODE REVIEW

File: pkg/storkctl/action.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	}
+
+	performCommands.AddCommand(
+		newFailoverCommand(cmdFactory, ioStreams),
+	)
+	return performCommands
+}
+
+func newFailoverCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	getClusterPairCommand := &cobra.Command{
```
Comment (adityadani) [inline pkg/storkctl/action.go:34]: nit: getFailoverCommand

File: pkg/resourceutils/updatereplicas.go [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+	if err != nil {
+		util.CheckErr(err)
+		return
+	}
+	configClient, err := k8sdynamic.NewForConfig(config)
+	if err != nil {
+		util.CheckErr(err)
+		return
+	}
+	ruleset := inflect.NewDefaultRuleset()
```
Comment (adityadani) [inline pkg/resourceutils/updatereplicas.go:110]: @diptiranjanpx Has a PR which moves this into a common function. Either one of you will have to rebase your changes

**Review by adityadani — APPROVED** [review]
lgtm

File: pkg/resourceutils/updatereplicas.go (hunk 2 above)
Comment (dgoel-px) [reply pkg/resourceutils/updatereplicas.go:110]: I will fix this in the horntail-failover branch

File: pkg/storkctl/action.go (hunk 1 above)
Comment (dgoel-px) [reply pkg/storkctl/action.go:34]: The convention used in the code is `newXXXCommand` everywhere, that's why I went with this

**Pull request merged**